### PR TITLE
support `scheduler_plugin_execution_duration_seconds` in scheduler_perf

### PR DESCRIPTION
**Pull request by sanposhiho**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

support `scheduler_plugin_execution_duration_seconds` in scheduler_perf, which visualizes which plugin is how much slower. 

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

I confirmed that it's working expectedly at local machine. The output would be like following.

```json
....
    {
      "data": {
        "Average": 0.00032820000000000006,
        "Perc50": 0.005,
        "Perc90": 0.009000000000000001,
        "Perc95": 0.0095,
        "Perc99": 0.0099
      },
      "unit": "ms",
      "labels": {
        "Metric": "scheduler_plugin_execution_duration_seconds",
        "Name": "SchedulingBasic/500Nodes/namespace-2",
        "extension_point": "Score",
        "plugin": "ImageLocality",
        "result": "not applicable"
      }
    },
    {
      "data": {
        "Average": 0.0047445000000000005,
        "Perc50": 0.005,
        "Perc90": 0.009000000000000001,
        "Perc95": 0.0095,
        "Perc99": 0.0099
      },
      "unit": "ms",
      "labels": {
        "Metric": "scheduler_plugin_execution_duration_seconds",
        "Name": "SchedulingBasic/500Nodes/namespace-2",
        "extension_point": "PreFilter",
        "plugin": "InterPodAffinity",
        "result": "not applicable"
      }
    },
.....
```

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by AxeZhan** [comment]
`uniqueLVCombos` looks a bit complex, I'm not sure if it's worth to maintain this function,
Considering that we now only have one metric("scheduler_plugin_execution_duration_seconds") that needs to be combined, and we can simply use two for loops to generate combinations for it.

But it is true that `uniqueLVCombos` can provide better abstraction, and I can't come up a better way if there will be more metrics with more labels added in the future :)

The rest part looks good to me and should be ok.

File: test/integration/scheduler_perf/scheduler_perf.go [hunk 1]
```diff
@@ -90,23 +91,65 @@ const (
 	configFile               = "config/performance-config.yaml"
 	extensionPointsLabelName = "extension_point"
 	resultLabelName          = "result"
+	pluginLabelName          = "plugin"
 )
 
 var (
 	defaultMetricsCollectorConfig = metricsCollectorConfig{
-		Metrics: map[string]*labelValues{
+		Metrics: map[string][]*labelValues{
```
Comment (utam0k) [inline test/integration/scheduler_perf/scheduler_perf.go:99]: I wonder if the metrics in scheduler-perf are under control like other metrics. It means we should list stable metrics.
https://kubernetes.io/docs/reference/instrumentation/metrics/#list-of-stable-kubernetes-metrics

WDYT? @logicalhan

Comment (utam0k) [reply test/integration/scheduler_perf/scheduler_perf.go:99]: This is a bit out of scope in this PR but let me ask you a question for our future.

Comment (logicalhan) [reply test/integration/scheduler_perf/scheduler_perf.go:99]: I don't think we're going to be able to parse this file..

Comment (logicalhan) [reply test/integration/scheduler_perf/scheduler_perf.go:99]: It looks... challenging from a static analysis perspective.

Comment (sanposhiho) [reply test/integration/scheduler_perf/scheduler_perf.go:99]: Probably I don't understand what @utam0k means. Why should we in the first place actually? 
All metrics used in scheduler_perf are existing metrics that are exposed from scheduler, not something like scheduler_perf's original metrics.

Comment (utam0k) [reply test/integration/scheduler_perf/scheduler_perf.go:99]: Sorry you are right.

**Review by utam0k — COMMENTED** [review]
/lgtm

**Pull request merged**